solr/solr-ref-guide/src/solr-on-hdfs.adoc

```diff
@@ -16,11 +16,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
-WARNING: Storing indexes in HDFS is deprecated and may be be removed in 9.0.
-This functionality may be moved to a 3rd-party plugin in the future.
 
 Solr has support for writing and reading its index and transaction log files to the HDFS distributed filesystem.
 
+Starting from 9.0, this functionality has been moved from Solr Core to the HDFS contrib module. 
```

Comment:
Not "contrib"

Reply:
Fixed in 9675773a7e4ebdd2a794a021db187576571d2db8